WordPress/gutenberg
# Add vw and vh units to the custom font size picker

## What?

Add the `vw` and `vh` units to the custom font size picker.
Partial for https://github.com/WordPress/gutenberg/issues/23323

## Why?


The custom font size picker is limited to using px, em, rem. Users are asking for the units to be extended (The units are also more limited than in the custom spacing control, where px, em, rem %, vw and vh are supported)

## How?

The PR updates the units in the font size picker component.

## Testing Instructions
Add a text based block like a paragraph.
Open the block settings and open the Typography panel.
Select "Set custom size". 
Confirm that vw and vh units can be selected.
Select them and confirm that the units are used in the editor and front.

### Testing Instructions for Keyboard


## Screenshots or screencast
